examples/step-2/doc/results.dox

```diff
@@ -73,3 +73,5 @@ Terminal type set to 'x11'
 gnuplot> set style data points
 gnuplot> plot "sparsity_pattern.1"
 @endcode
+
+Another practice based on <ahref="http://www.gnuplot.info/">GNUPLOT</a> is trying to print out the mesh with locations and numberings of the support points where you might need to use GridOut and MappingQ1.
```

Comment:
Can you break this line to roughly 80 characters? You will also have to put a space between `<a` and `href`. There may be more things I can't currently see because they scroll out to the right :-)

Reply:
Sorry about about the long line, I just divided it into several short lines, hope it will work. Also added the code in the end.